setup/azurecmdfiles.wxi

```diff
@@ -3984,9 +3984,6 @@
                         <Component Id="cmpF7ADD37722F45985B5323202F35958F2" Guid="*">
                             <File Id="fil0FA91D9C31AF07DCD03BFB842FE2B77E" KeyPath="yes" Source="$(var.sourceDir)\ServiceManagement\Azure\Compute\Microsoft.WindowsAzure.Commands.ServiceManagement.PlatformImageRepository.dll" />
                         </Component>
-                        <Component Id="cmp9D64A1111722F25BC100121E73BE039E" Guid="*">
```

Comment:
@neetus15 I checked it on my side with a fresh clone, with this set of given changes, all the changes that are now shown in setup should not be there.
One possibility is:
1) You have not pulled from upstream in a while (if not do a pull and get the latest changes)
2) rebuild again and then build the installer using generate.ps1
Remember, all you should see with your set of changes is an additional dll getting added to the setup.
In a nutshell if you diff the output directory (\azure-powershell\src\Package\Debug\ServiceManagement) prior to your change and after your change, the only difference should be an additional .dll, that is exactly what setup update should be (unless you have added additional resource files or other files that needs to be installed on the users machines)
But apart from those changes, no files should be deleted or nothing additional should be added.